src/components/breadcrumbs/BreadcrumbLink.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import {
+  chakra,
+  BreadcrumbLink as ChakraBreadcrumbLink,
+  BreadcrumbLinkProps as ChakraBreadcrumbLinkProps,
+  useMultiStyleConfig,
+} from '@chakra-ui/react';
+
+interface BreadcrumbLinkProps extends ChakraBreadcrumbLinkProps {
+  isLast?: boolean;
+}
+
+const BreadcrumbLink: FC<PropsWithChildren<BreadcrumbLinkProps>> = ({
+  children,
+  ...itemProps
+}) => {
+  const styles = useMultiStyleConfig('Breadcrumbs');
+  const { isLast } = itemProps;
+
+  return isLast ? (
```

Comment:
I would assume the logic here is more if there is no link do not render it as a link rather than if it's the last item this might be the case for the concrete use case you have in the PDP

Reply:
Wasn't sure how the data is going to be delivered for the breadcrumbs, so it kinda made sense to me to build it in that way since it's always going to work, no matter if it has url or not.

Updated tho.